hack/jenkins/test-image/run.sh

```diff
@@ -42,6 +38,10 @@ export KUBE_COVERPROCS=8
 export KUBE_INTEGRATION_TEST_MAX_CONCURRENCY=4
 export LOG_LEVEL=4
 
+./hack/build-go.sh
+godep go install ./...
+./hack/install-etcd.sh
```

Comment:
Unfortunately #15336 is only one of several changes in test scripts between 1.0/1.1 and head. We can either go through and cherrypick all the changes, or we can have a branch-specific `hack/jenkins/test-image/run.sh` for each branch that runs the version of verify* scripts that exist on the branch.